Src/AutoFixture/CustomizeAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Reflection;
+
+namespace Ploeh.AutoFixture
+{
+    /// <summary>
+    /// Base class for customizing parameters in methods.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Parameter, AllowMultiple = true)]
+    public abstract class CustomizeAttribute : Attribute
```

Comment:
That's a good point; I certainly wouldn't mind moving towards [Passive Attributes](http://blog.ploeh.dk/2014/06/13/passive-attributes) if possible. Normally, however, Passive Attributes rely on an extensible filtering mechanism, like I've described in the linked article. Otherwise, users can't plug in their own Passive Attributes.

So far, the unit testing frameworks targeted by AutoFixture don't have such a filtering mechanism. In both xUnit.net and NUnit, the extensibility points seem to revolve around Active Attributes. I must admit, though, that I haven't studied the extensibility mechanism of either xUnit.net 2 or NUnit 3 enough to be sure. This is certainly an area where I'd love to be proven wrong.